lib/sms_notificaiton.rb

```diff
@@ -0,0 +1,23 @@
+require 'twilio-ruby'
+
+
+class SmsNotification
+def order
+account_sid = "REMOVED"
+auth_token = "REMOVED"
+
+delivery_time = Time.now + 3600
+
+begin
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    @client.account.messages.create({
+        :from => 'REMOVED',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)